fulp_modules/devils/devil.dm

```diff
@@ -0,0 +1,374 @@
+#define DEVIL_BODYPART "devil"
+#define DEVIL_HANDS_LAYER 1
+#define DEVIL_TOTAL_LAYERS 2
+
+/mob/living/carbon/true_devil
+	name = "True Devil"
+	desc = "A pile of infernal energy, taking a vaguely humanoid form."
+	icon = 'fulp_modules/devils/devil_icon/devils.dmi'
+	icon_state = "true_devil"
+	gender = NEUTER
+	health = 350
+	maxHealth = 350
+	ventcrawler = VENTCRAWLER_NONE
+	density = TRUE
+	pass_flags =  0
+	sight = (SEE_TURFS | SEE_OBJS)
+	status_flags = CANPUSH
+	mob_size = MOB_SIZE_LARGE
+	held_items = list(null, null)
+	bodyparts = list(
+		/obj/item/bodypart/chest/devil,
+		/obj/item/bodypart/head/devil,
+		/obj/item/bodypart/l_arm/devil,
+		/obj/item/bodypart/r_arm/devil,
+		/obj/item/bodypart/r_leg/devil,
+		/obj/item/bodypart/l_leg/devil,
+		)
+	hud_type = /datum/hud/devil
+	var/ascended = FALSE
+	var/mob/living/oldform
+	var/list/devil_overlays[DEVIL_TOTAL_LAYERS]
+
+/// Arch devil
+/mob/living/carbon/true_devil/arch_devil
+	health = 500
+	maxHealth = 500
+	ascended = TRUE
+	icon_state = "arch_devil"
+
+/mob/living/carbon/true_devil/Initialize()
+	create_bodyparts()
+	create_internal_organs()
+	grant_all_languages()
+	. = ..()
+	ADD_TRAIT(src, TRAIT_SPACEWALK, INNATE_TRAIT)
+	ADD_TRAIT(src, TRAIT_ADVANCEDTOOLUSER, INNATE_TRAIT)
+	ADD_TRAIT(src, TRAIT_NOBREATH, INNATE_TRAIT)
+	ADD_TRAIT(src, TRAIT_NEVER_WOUNDED, INNATE_TRAIT)
+	AddSpell(new /obj/effect/proc_holder/spell/targeted/conjure_item/summon_pitchfork(null))
+	AddSpell(new /obj/effect/proc_holder/spell/targeted/conjure_item/violin(null))
+	AddSpell(new /obj/effect/proc_holder/spell/aimed/fireball/hellish(null))
+	AddSpell(new /obj/effect/proc_holder/spell/targeted/summon_dancefloor(null))
+
+/mob/living/carbon/true_devil/create_internal_organs()
+	internal_organs += new /obj/item/organ/brain
+	internal_organs += new /obj/item/organ/tongue
+	internal_organs += new /obj/item/organ/eyes/robotic/shield
```

Comment:
bruuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuh

you can emp these (and/or extract them and be confused as to why all devils are part-robot?). just make an invincible eye subtype like you did with the ears.